finance-api/user-guides/fetch-account-transactions.md

```diff
@@ -0,0 +1,136 @@
+Fetch account transactions
+===
+Using the Finance API, you can fetch transactions or transactions of certain types from a merchant's Zettle liquid account during a specific period.
+
+> **Note**: A merchant's Zettle liquid account contains all confirmed transactions that are already paid out or to be paid out to the merchant. If you want to check preliminary transactions that are being confirmed with the issuing banks (buyers' banks), you can fetch them from a merchant's Zettle preliminary account.
+
+<!-- Is there any limit for how old transactions can be fetched? -->  
+
+* [Prerequisites](#prerequisites)
+* [Fetch transactions during a specific period](#fetch-transactions-during-a-specific-period)
+* [Fetch transactions of certain types during a specific period](#fetch-transactions-of-certain-types-during-a-specific-period)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up with the required OAuth scope using [Authorization OAuth2 API](../../authorization.adoc). 
+
+## Fetch transactions during a specific period
+To fetch transactions from a merchant's Zettle account during a specific period, you need to fetch them from their Zettle liquid account.
+<!-- Check with team Ledger: Is it correct that a PAYMENT transaction can have a PAYOUT state? When I fetched all transactions, I got transactions that include PAYOUT. How can the integrator tell which PAYOUT is from which transaction type? --> 
+
+1. Fetch transactions from the merchant's Zettle liquid account with pagination.
+     
+   ```
+   GET /organizations/self/accounts/{accountTypeGroup}/transactions?start={start_time}&end={end_time}&{limit}&{offset}
+   ```
+
+   Example:
+   
+   The following example fetches all transactions from the merchant's Zettle liquid account from 1 January, 2020 to 31 December, 2020. The requests fetch three transactions at a time, with pagination set by `limit` and `offset`. 
+   
+   Request
+   
+   ```
+   GET /organizations/self/accounts/liquid/transactions?start=2020-01-01&end=2020-12-31&limit=3&offset=0
+   ```
+   ```
+   GET /organizations/self/accounts/liquid/transactions?start=2020-01-01&end=2020-12-31&limit=3&offset=3
+   ```
+   Response
```

Comment:
How come one response for two requests?
And if `limit` is 3  why there are 6+ transactions in the response?